### DISCUSSION TITLE
Airtronics/Sanwa FHSS Protocols?

### DISCUSSION THREAD

**Discussion by steveveach** [pascallanger/DIY-Multiprotocol-TX-Module#736] [Ideas]
Hello everyone. I'm new to this forum and new to EdgeTX. I own the RadioMaster TX16S Mark II Max with the multi-protocol RF module internally which they call the 4-in-1 module.

My brother would love to buy a new EdgeTX transmitter but he would need it to support the Airtronics receivers in his more than 20 airplanes. It seems that the current versions of EdgeTX (and OpenTX) don't support the Airtronics/Sanwa FHSS protocols. I would imagine there are thousands of these receivers still in use.

Would it be possible for someone to reverse engineer the Airtronics receiver/transmitter if my brother was willing to loan them for analysis. I don't know if this request should be aimed at the multi-protocol team, the EdgeTX team, the OpenTX or all of them.

Thanks!

**Comment by pascallanger** [comment]
Hi, I would need a TX and RX to be shipped to my attention in France to see what's feasible.

**Reply by steveveach** [reply]
Please help me understand the process. If you were able to determine what's feasible, would that ultimately be a firmware update for the RF multi-protocol module that's built inside my transmitter? Would EdgeTX need to be updated as well to access the new protocols? I'm new to all this and don't understand the subtleties of how this all gets developed and works together. Thank you.
